deep_qa/training/trainer.py

```diff
@@ -591,7 +606,11 @@ def _save_auxiliary_files(self):
         Called after training. If you have some auxiliary object, such as an object storing
         the vocabulary of your model, you can save it here. The model config is saved by default.
         """
-        model_config = self.model.to_json()
+        if self.num_gpus > 1:
+            lambda_layers = len(self.model.outputs)
```

Comment:
`num_lambda_layers` - when I see a variable named `lambda_layers`, I think it's a list.